provider/src/main/java/com/synopsys/integration/alert/provider/blackduck/action/BlackDuckDistributionApiAction.java

```diff
@@ -85,14 +92,22 @@ private void afterWrite(FieldModel currentFieldModel) throws AlertException {
         }
     }
 
-    private Collection<String> extractConfiguredProjects(FieldModel currentFieldModel) {
-        return currentFieldModel.getFieldValueModel(ProviderDistributionUIConfig.KEY_CONFIGURED_PROJECT)
-                   .map(FieldValueModel::getValues)
-                   .orElse(Set.of());
+    private Collection<JobProviderProjectFieldModel> extractConfiguredProjects(FieldModel currentFieldModel) {
+        Collection<String> providerProjectJsonStrings = currentFieldModel.getFieldValueModel(ProviderDistributionUIConfig.KEY_CONFIGURED_PROJECT)
```

Comment:
The configured projects are not a list of project names. They are a list of JSON strings.